theforeman.org/pipelines/release/source/smart-proxy.groovy

```diff
@@ -9,14 +9,27 @@ pipeline {
     }
 
     stages {
-        stage("Clone repository") {
-            steps {
-                git url: git_url, branch: git_ref
-            }
-        }
-        stage("test ruby") {
-            steps {
-                run_test(ruby: '2.7', '3.0', '3.1')
+        stage('Test') {
+            matrix {
+                agent any
+                axes {
+                    axis {
+                        name 'ruby'
+                        values '2.7', '3.0', '3.1'
+                    }
+                }
+                stages {
+                    stage("Clone repository") {
+                        steps {
+                            git url: git_url, branch: git_ref
+                        }
+                    }
+                    stage("Test Ruby") {
+                        steps {
+                            run_test(ruby)
```

Comment:
It doesn't find this variable and I wonder why. Looking at https://www.jenkins.io/doc/book/pipeline/syntax/#declarative-matrix I think it should be
```suggestion
                            run_test(env.ruby)
```

Though the docs aren't very clear on it.

Reply:
@ekohl i guess i've created some git mess here, can you help?
here is the change: https://github.com/theforeman/jenkins-jobs/pull/326